lib/pages/scout/DesktopScout.dart

```diff
@@ -565,23 +563,12 @@ class _DesktopScoutState extends State<DesktopScout> {
         ]);
   }
 
-  Widget buildListview() {
+  Widget buildListview(ScoutPageState state) {
     double _height = MediaQuery.of(context).size.height;
     double hgt = _height * 0.8 - 120;
-
-    if (nflList.length == 0) {
-      nflList = AthleteList.list;
-      nflListFilter = nflList;
-    }
-
-    if (allList.length == 0) {
-      // Filter all the athletes. For now we only have NFL athletes
-      allList = nflList;
-      allListFilter = allList;
-    }
-
+    List<AthleteScoutModel> athletesList = state.athletes;
     // all athletes
-    if (sportState == 0)
+    if (state.selectedSport == SelectedSport.ALL)
```

Comment:
This is a lot of repetitive code... the sport selection should not need to be queried the bloc will send exactly what needs to be shown in the list so there is no need for any branching logic